### ISSUE TITLE
Mutual dependence on cinnamon packages.

### ISSUE THREAD

**Issue by ghost** [linuxmint/Cinnamon#3235]
I have created a repository. Just for now testing some things about Cinnamon in ubuntu . And also created a script in python to automate all this and that more people can have their own repo in Ubuntu cinnamon. I know at least two "distros" that try to ubuntu + cinnamon. One of them could also use Mint distro, but did not like how the  packages are handles on Mint.

The fact is that Ubuntu user can not know when Mint can decide remove also the test repo of Cinnamon. Ubuntu can go in a direction opposite to the direction of Mint, and this can be the cause of remove the unstable repo.

I think it would be good and convenient that other external repositories in Ubuntu will be emerged to install Cinnamon.

The fact that many more distributions with Cinnamon desktop appear, it shows that the work has been done has a good worthwhile. Although not a necessary sample, because the good developers like all of you, always know when something is good .

In short, do not know if this is a problem or not, but the  cinnamon package and cinnamon-screensaver package have dependencies between them(a cycle/recursive dependency). Also the same thing happens with the nemo package and the cinnamon package. An alternative could be sought for resolve this recursive loop. From a software design, should always exist a better way to solve a recursive dependency.

https://launchpad.net/~lestcape/+archive/cinnamon/+packages
https://github.com/lestcape/Cinnamon-Ubuntu-Installer

Thank you.
